The android-x86 build started failing, but I don't think it can have anything to do with this PR. Plus we are not using android at all and all other targets are fine. Here's the error, it looks like the ASTC encoder is trying to use a x64 symbol in the x86 build. Maybe the NDK the job pulls has been updated or something like that....
`../../../../bimg/3rdparty/astc-encoder/source/astcenc_vecmathlib_sse_4.h:1281:26: error: use of undeclared identifier '_mm_popcnt_u64'; did you mean '_mm_popcnt_u32'?`

Edit: in fact, they had the same error in the main bgfx repo and they disabled all android targets except arm64: https://github.com/bkaradzic/bgfx/commit/14b82682991b09ebfa626ee43dcd913f0c67f220.

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -211,10 +211,7 @@ jobs:
       fail-fast: true
       matrix:
         include: [
-          { platform: arm    },
           { platform: arm64  },
-          { platform: x86    },
-          { platform: x86_64 },
         ]
     name: android-${{ matrix.platform }}
     runs-on: ubuntu-22.04

